mods/d2k/rules/defaults.yaml

```diff
@@ -225,6 +227,11 @@
 			Rough: 80
 	Voiced:
 		VoiceSet: InfantryVoice
+	WithParachute:
+		UpgradeTypes: parachute
+		UpgradeMinEnabledLevel: 1
+		ShadowImage: parach-shadow
+		ShadowSequence: idle
```

Comment:
Altho, I don't remember messing around with this particular SHP. It was just a copy paste if I remember correctly